### DISCUSSION TITLE
Launching qmdr

### DISCUSSION THREAD

**Discussion by sheasbys** [hmatuschek/qdmr#253] [Q&A] [answered]
I am receiving an Anytone 878 today and was hoping to get the app running in the meantime. Being tech savvy, I am embarrassed to ask such a noob question about launching the application. I assumed you run it from the CLI by simply typing qmdr but I get a command not found, and yes, I have installed it via MacPorts successfully. 

Any ideas would be appreciated.

Stephen, WO0KII

**Comment by hmatuschek** [comment, marked as answer]
Actually, there should be an app package installed along with qdmr. See https://github.com/macports/macports-ports/blob/9c275ebac87afe22a30ef6aa62057b4836c832b0/science/qdmr/Portfile#L48 

That is, CMD+Space -> enter qDMR (case insensitive) should do it or find it in the launchpad (or however the overview of applications was called).

**Reply by sheasbys** [reply]
One tweak to your instructions is to do a spotlight search (command + Space) and enter qMDR.app and it will launch the app. To create a shortcut, the app is located here: /Applications/MacPorts/qMDR.app.

**Reply by hmatuschek** [reply]
Thanks for correcting it. I'm sitting in a train on the way home at a Linux box. So I couldn't look it up. I am, however, surprised, that the shortcut is not created automatically. Maybe there is a MacPorts group and under there, you find the App bundle (which is just a link, pointing to the actual executable normally located under /opt/local/bin).

**Reply by sheasbys** [reply]
I just could not find it and did not think of looking in the MacPorts folder.

**Reply by hmatuschek** [reply]
This is kind of unintuitive, one expects the MacPorts folder to contain apps related to MacPorts itself, not independent software installed by MacPorts. Anyway, I am glad that it works.

**Reply by sheasbys** [reply]
Agreed, this is not intuitive. Appreciate your help and your hard work on this.

**Reply by hmatuschek** [reply]
Thanks, helps to keep the motivation up.
